packages/gatsby-remark-code-repls/README.md

```diff
@@ -0,0 +1,42 @@
+# gatsby-remark-code-repls
+
+Gatsby plugin to auto-generate links to popular REPLs like [Babel](https://babeljs.io/repl/) and [Codepen](https://codepen.io/).
+
```

Comment:
Perhaps add more of your PR discussion here? Would be good to help people running across the plugin to understand the problem it's solving.

Reply:
Done.